ensemble-view/src/main/java/hec/ensembleview/EnsembleChartAcrossTime.java

```diff
@@ -68,6 +71,41 @@ public void addLine(LineSpec line) throws ParseException {
 
     @Override
     public ChartPanel generateChart() {
+        XYPlot plot = createTimeSeriesPlot();
+
+        ChartPanel chart = new ChartPanel(new JFreeChart(chartTitle, plot)){
+            @Override
+            public void mousePressed(MouseEvent e)
```

Comment:
A few of the modifiers in this method are deprecated. Is it possible to use the recommended modifiers instead?

Reply:
We will be changing the class entirely from JFreeChart to gfx2D

Reply:
As well as the EnsembleChartAcrossEnsembles class